apps/portals/src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -393,44 +348,17 @@ const routes: GenericRoute[] = [
       {
         exact: true,
         path: 'Data',
-        synapseConfigArray: [
-          {
-            name: 'RouteControlWrapper',
-            isOutsideContainer: true,
-            props: {
-              ...RouteControlWrapperProps,
-              synapseConfig: data,
-            },
-          },
-        ],
+        hideRouteFromNavbar: false,
       },
       {
         exact: true,
         path: 'Publications',
-        synapseConfigArray: [
-          {
-            name: 'RouteControlWrapper',
-            isOutsideContainer: true,
-            props: {
-              ...RouteControlWrapperProps,
-              synapseConfig: publications,
-            },
-          },
-        ],
+        hideRouteFromNavbar: false,
```

Comment:
The `hideRouteFromNavBar` is not strictly necessary, but I hope it makes it more clear that this route is necessary to ensure we get a nav menu entry for this Explore page--the current nav menu code doesn't inspect the `RouteControlWrapper` internal routes.